Pod/Classes/UI/CardTextField.swift

```diff
@@ -283,6 +309,18 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         monthTextField?.deleteBackwardCallback = {_ -> Void in self.numberInputTextField?.becomeFirstResponder()}
         yearTextField?.deleteBackwardCallback = {_ -> Void in self.monthTextField?.becomeFirstResponder()}
         
+        if isRightToLeftLanguage {
+            cvcTextField.textAlignment = .Left
+            monthTextField.textAlignment = .Left
+            yearTextField.textAlignment = .Right
+            numberInputTextField.textAlignment = .Right
+        } else {
+            cvcTextField.textAlignment = .Right
+            monthTextField.textAlignment = .Right
+            yearTextField.textAlignment = .Left
+            numberInputTextField.textAlignment = .Left
```

Comment:
Unfortunately the `Natural` option does not have an equivalent for `Right`. I could have used Natural for `yearTextField` and `numberInputTextField` but thought this implementation is more descriptive of what is happening.

Ideally, I would have gone with `Natural` and an `NaturalInverted` - only that the latter does not exist.